core/controllers/admin.py

```diff
@@ -631,3 +632,17 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class SendDummyMailHandler(base.BaseHandler):
+    """This function handles sending test emails."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        response = {
+            'msg': 'Success! Mail sent',
+        }
+        email_manager.send_dummy_mail(feconf.ADMIN_EMAIL_ADDRESS)
```

Comment:
send_dummy_mail_to_admin() and pass no argument

Reply:
Done